presto-hive-hadoop2/bin/run_hive_tests.sh

```diff
@@ -7,6 +7,11 @@ set -euo pipefail -x
 cleanup_docker_containers
 start_docker_containers
 
+# restart HMS to pickup memory settings
+exec_in_hadoop_master_container cp /etc/hadoop/conf/hive-env.sh /etc/hive/conf/hive-env.sh
```

Comment:
Ideally we should move docker related changes to the Docker definition itself. But let's not block this PR by that. Let's have a separate PR addressing that.